async_firebase/base.py

```diff
@@ -71,6 +62,22 @@ def __init__(
 
         self._request_timeout = request_timeout
         self._request_limits = request_limits
+        self._http_client: t.Optional[httpx.AsyncClient] = None
+
+    @property
+    def _client(self) -> httpx.AsyncClient:
+        def _create_http_client() -> httpx.AsyncClient:
+            return httpx.AsyncClient(
+                timeout=httpx.Timeout(**self._request_timeout.__dict__),
+                limits=httpx.Limits(**self._request_limits.__dict__),
+            )
+
+        if self._http_client is None:
```

Comment:
Maybe we could have a single line with these conditions `if self._http_client is None or self._http_client.is_closed`

Reply:
Fixed.